src/WebJobs.Script.WebHost/WebJobsScriptHostService.cs

```diff
@@ -378,6 +382,11 @@ private async Task UnsynchronizedStartHostAsync(ScriptHostStartupOperation activ
                     }
                 }
 
+                if (_workerConfigResolverOptionsChangeTokenSource is HostBuiltChangeTokenSource<WorkerConfigurationResolverOptions> { } hostBuiltChangeToken)
+                {
+                    hostBuiltChangeToken.TriggerChange();
+                }
```

Comment:
Lines 385-388 are implemented similarly to lines 390-393

Reply:
I filed https://github.com/Azure/azure-functions-host/issues/11208 to make this easier in the future.